ogr/ogrsf_frmts/gml/parsexsd.h

```diff
@@ -38,6 +38,6 @@
 
 bool CPL_DLL GMLParseXSD(const char *pszFile,
                          std::vector<GMLFeatureClass *> &aosClasses,
-                         bool &bFullyUnderstood);
+                         bool &bFullyUnderstood, bool &useSchemaImports);
```

Comment:
useSchemaImports is an *input* parameter, so it should be passed  by value, and not reference.
And the logic we apply in argument ordering is to put input parameters first, and output ones afterwards, so that should be:

bool CPL_DLL GMLParseXSD(const char *pszFile,
                         bool bUseSchemaImports,
                         std::vector<GMLFeatureClass *> &aosClasses,
                         bool &bFullyUnderstood);

Please also use the 'b' prefixing to indicated boolean, to be consistent of the convention applied usually in GDAL. So bUseSchemaImports (here and in other places)

Reply:
Thank you for the explanation, I have fixed it.